ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/logic.j2

```diff
@@ -1,6 +1,6 @@
 {% for connected_endpoints_key in connected_endpoints_keys %}
 {%     set switch_connected_endpoints = hostvars[inventory_hostname][connected_endpoints_key] | arista.avd.default({}) %}
-{%     do connected_endpoints.update({connected_endpoints_key: {}})%}
+{%     do connected_endpoints.update({connected_endpoints_key: {} }) %}
```

Comment:
IMO, there should be a space before connected_endpoints_key as well. But that's probably something to fix manually, not with linter.

Reply:
@ankudinov that depends. Pep8 says no spacing.

Reply:
@ankudinov Done. Please take a look.